packages/python/src/symbols/python-output-symbol.ts

```diff
@@ -2,17 +2,46 @@ import {
   Children,
   OutputSymbol,
   OutputSymbolOptions,
-  refkey,
-  useBinder,
-  useContext,
-  useScope,
+  track,
+  TrackOpTypes,
+  trigger,
+  TriggerOpTypes,
 } from "@alloy-js/core";
-import { SourceFileContext } from "../components/SourceFile.js";
+
+export interface CreatePythonSymbolOptions extends OutputSymbolOptions {
+  module?: string;
+  children?: Children;
```

Comment:
I looked at the `PythonOutputSymbol` creation and don't see the usage of `children`. Shouldn't the container vs children be managed by scopes rather than embedding children in the `PythonOutputSymbol`?

Reply:
TBH I am not sure why that's there, maybe it was part of one of my earlier iterations. Removing it.